sync/sync.go

```diff
@@ -23,19 +23,21 @@ type Sync interface {
 	Fetch(path string) (*Node, error)
 	Update(path, json string) error
 	Delete(path string) error
+	WatchEx(path string, responseChan chan *Event, stopChan chan bool, revision int64) error
```

Comment:
if the only difference is revision parameter, it's better to update existing `Watch` method instead of creating new one.